python/hail/context.py

```diff
@@ -724,6 +724,40 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, min_p
         return VariantDataset(self, jvds)
 
     @handle_py4j
+    @record_method
+    @typecheck_method(path=oneof(strlike, listof(strlike)),
+                      min_partitions=nullable(integral),
+                      drop_samples=bool,
+                      cell_type=strlike)
+    def import_matrix(self, path, min_partitions=None, drop_samples=False, cell_type="Int64", missing="NA"):
+        """
+        :param path: File(s) to read. Currently, takes 1 header line of column ids and subsequent lines of rowID, data... in TSV form where data can be parsed as an integer.
+        :type path: str or list of str
+
+        :param min_partitions: Number of partitions.
+        :type min_partitions: int or None
+
+        :param bool drop_samples: I don't know if this is relevant, but it only loads the row IDs.
+
+        :param str cell_type: Tells function how to parse cell data. Can be Int32, Int64, Float32, Float64, or String. Default: Int64
+
+        :param str missing: notation for cell with missing value. Default: "NA"
+
+        :return: Variant dataset imported from file(s)
+        :rtype: :py:class:`.VariantDataset`
+        """
+        tdict = {"Int32": 0, "Int64": 1, "Float32": 2, "Float64": 3, "String": 4}
```

Comment:
Unnecessary.  You can pass the type object directly to loadMatrices by doing `cell_type._jtype`.